deno/payloads/v10/_interactions/_applicationCommands/_chatInput/base.ts

```diff
@@ -1,9 +1,12 @@
+import type { LocaleString } from '../../../../../v10.ts';
 import type { APIApplicationCommandOptionChoice, ApplicationCommandOptionType } from './shared.ts';
 
 export interface APIApplicationCommandOptionBase<Type extends ApplicationCommandOptionType> {
 	type: Type;
 	name: string;
+	name_localizations?: Record<LocaleString, string>;
```

Comment:
All of these should be Partial'd (as not all locales have to be defined/are defined)